File: tests/Pieces/EagerTest/java/FiveMultiLevelIndirectInvocations/src/FooTest.java
import org.junit.TestCase;

public class FooTest extends TestCase {

	Bar bar;
	Baz baz;
	Fal fal;
	Foo foo;
	Fuu fuu;

	void helperLvl1() {
		helperLvl2();
	}

	void helperLvl2() {
		helperLvl3();
	}

	void helperLvl3() {
		bar.bar();
		baz.baz();
		fal.fal();
		foo.foo();
		fuu.fuu();
	}

	public void testFoo() {
		helperLvl3();
	}
}

File: tests/Pieces/VerboseTest/cpp/SLOCTresholdExceeded/src/FooTest.cpp
#include <cppunit/TestCase.h>
#include "UUT.h"

using CppUnit::TestCase;

class FooTest : public TestCase {
public:
	void testFoo();
private:
	UUT uut;
};

void FooTest::testFoo() {
	uut.foo();
	uut.fal();
	uut.bar();
	uut.baz();
	uut.git();

	uut.foo();
	uut.fal();
	uut.bar();
	uut.baz();
	uut.git();

	uut.foo();
	uut.fal();
	uut.bar();
	uut.baz();
	uut.git();

	uut.foo();
	uut.fal();
	uut.bar();
	uut.baz();
	uut.git();
}
